app/src/main/java/com/texthip/thip/ui/group/makeroom/component/GroupRoomDurationPicker.kt

```
@@ -0,0 +1,248 @@
+        if (selectedDate.isBefore(today)) {
+            startYear = today.year
+            startMonth = today.monthValue
+            startDay = today.dayOfMonth
+        }
+    }
+
+    // 날짜 객체로 변환
+    val startDate = remember(startYear, startMonth, startDay) {
+        try {
+            LocalDate.of(startYear, startMonth, startDay)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)